wagtail_localize/machine_translators/deepl.py

```diff
@@ -17,8 +27,11 @@ class DeepLTranslator(BaseMachineTranslator):
     display_name = "DeepL"
 
     def translate(self, source_locale, target_locale, strings):
+        api_endpoint = "https://api.deepl.com/v2/translate"
+        if "API_ENDPOINT" in self.options:
+            api_endpoint = self.options["API_ENDPOINT"]
```

Comment:
suggestion:

```suggestion
        api_endpoint = self.options.get("API_ENDPOINT", "https://api.deepl.com/v2/translate")
```

Reply:
@zerolab thanks for the feedback, I'll try to get it done asap.